gear-programs/vft-manager/app/src/services/mod.rs

```diff
@@ -373,7 +380,14 @@ impl VftManager {
     ) -> Result<(), Error> {
         self.ensure_running()?;
 
-        submit_receipt::submit_receipt(self, slot, transaction_index, receipt_rlp).await
+        submit_receipt::submit_receipt(self, slot, transaction_index, receipt_rlp)
+            .await
+            .map(|_| {
+                let _ = self.emit_event(Event::RlpReceiptProcessed {
```

Comment:
We need to emit event with info about ERC20/VF tokens transfer. Similar to `BridgingAccepted ` on Ethereum side (as an example - https://holesky.etherscan.io/tx/0x5b90c7b3b6e72201506b70af6c9e99d6c5ed864b8740bc17335d497a31b642b6#eventlog).

So it should look like this:
```rust
struct BridgingAccepted {
     to: ActorId,
     // who sent it on Eth side
     from: H160,
     amount: U256,
     // Vara token Id
     token: ActorId,
};
```

All the data are contained by the `BridgingRequested` event so we need to pass it to the token operations funcs and then to the handle_reply to emit event here since the reply func is benched. However we don't have access to the program/service so just send gasless message to zero address.